lib/client.js

```diff
@@ -8,7 +8,7 @@
 
 // export { client as supabase };
 
-import { createClient } from 'https://cors-anywhere.herokuapp.com/https://esm.sh/@supabase/supabase-js@2'
+import { createClient } from 'https://esm.sh/@supabase/supabase-js@2'
```

Comment:
No, actually it wasn't working with npm. But it's working well with CDN. However, earlier we used that hack of adding `https://cors-anywhere.herokuapp.com/` to prevent cors error & we used to open the demo link alongside, this particular arrangement isn't required now. It's working fine with regular CDN now.